rfcs/text/0003-object-field.md

```diff
@@ -1,30 +1,33 @@
 -->
-Numerous SaaS Cloud Providers provide event data where the event reflects an action (Typically a CRUD action) on what can broadly be considered an "object." For some providers that Object may be a "file" wherein the existing ECS `file` fields are adequate; however other providers (Workday, Salesforce, Service Now, Zoom, etc.) an object can represent some other concept like a "record" or "setting" or "meeting." There presently isn't any location within ECS to document these as there is with "file," forcing any representation to be pushed into a custom field. This RFC proposes a new top-level field called `object` that will represent these instances.
+Numerous SaaS and enterprise content/productivity services provide event data where the event reflects an action (typically `create`, `read`, `update`, `delete`) on what can broadly be considered items/documents/resources/records (originally dubbed "objects"). The are generally considered audit logs, or business analytics events. For some of these services, parts of (or in some cases, the entirety of) the corpus consists of _files_ (in the filesystem sense of the term) wherein the existing ECS `file` fields are adequate. However, most enterprise collaboration, storage, and communication systems (Workday, Salesforce, ServiceNow, Zoom, GitHub, G Suite, Zendesk, Jira, Confluence - Cloud - Server, etc.) present data units as "records", "documents", "tickets", "meetings" and more. ECS doesn't currently account for these entities, and relies on `file` as an option, albeit semantically incorrect and incomplete for the purpose of event tracking.
 
 ## Fields
 
 <!--
 Stage: 1: Describe at a high level how this change affects fields. Which fieldsets will be impacted? How many fields overall? Are we primarily adding fields, removing fields, or changing existing fields? The goal here is to understand the fundamental technical implications and likely extent of these changes. ~2-5 sentences.
 -->
-This RFC calls for the introduction of a top-level `object` field with an initial fieldset of the following child fields:
+This RFC calls for the introduction of a top-level `enterprise_content` field with an initial fieldset of the following child fields:
 
 | field | type | description |
 | --- | --- | --- |
-| `object.name` | keyword | Name of the Object |
-| `object.id` | keyword | ID of the Object |
-| `object.type` | keyword | Type of Object Represented (e.g. `record`, `meeting`, `repository`, `organization`, etc.) |
-| `object.additional_details` | object | Custom Key/Value pairs representing data relevant to the object |
-| `object.owner.id` | keyword | ID of the Object Owner |
-| `object.owner.name` | keyword | Name of the Object Owner |
-| `object.owner.email` | keyword | Email of the Object Owner |
+| `enterprise_content.document.name` | keyword | Name of the Enterprise Content Object |
```

Comment:
```suggestion
| `enterprise_content.document.name` | wildcard | Name of the Enterprise Content Object |
```

Multiple `.name` fields (`process.name`, `organization.name`, etc.) are being targeted for the initial [wildcard adoption](https://github.com/elastic/ecs/blob/master/rfcs/text/0001-wildcard-data-type.md).

We can discuss more once we have example field values to compare against, but `wildcard` could make sense as the type here as well.

Reply:
👍  FYI @ioanatia